lib/models/Team.js

```diff
@@ -0,0 +1,23 @@
+const pool = require('../utils/pool');
+
+module.exports = class Team {
+  id;
+  name;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.founded = row.founded;
+    this.founder = row.founder;
+  }
```

Comment:
Don't forget to also have founded and founder variables stated at the top of the file with id and name.